### PR TITLE
Feature/32 profile page main info

### PR DESCRIPTION
**Pull request by SashaBondarchuk**
https://trello.com/c/jQBJeuH5/32-profile-page-main-info

- added UI of profile page main info
- refactored color variables and removed dublicating throughout the whole project

image

### CODE REVIEW

File: frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<div class="profile-main-info-wrapper d-flex flex-column">
+
+    <div class="profile-main-info-header info-container">
+        <div class="user-info-container d-flex flex-row">
+            <div class="user-avatar-container">
+                <img src="../../../../assets/images/default-profile-img.jpg" alt="User avatar" class="user-avatar">
```
Comment (tatianahutii) [inline frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.html]:
```suggestion
                <img src="/assets/images/default-profile-img.jpg" alt="User avatar" class="user-avatar">
```

File: frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            </div>
+            <div class="user-info d-flex flex-column">
+                <p class="username">TatiHutii</p>
+                <p class="user-rank"><span>Rank</span>~5,000,000</p>
```
Comment (tatianahutii) [inline frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.html:10]: Please make all items not hard coded

We should show all data from our interfaces

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.html (hunk 2 above)
Comment (SashaBondarchuk) [reply frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.html:10]: Do you mean to create real interfaces? or just make it like 
`<p class="username">{{user.username}}</p>`
 `<p class="user-rank"><span>Rank ~</span>{{user.rank}}</p>`
and in component.ts fill that data with fake user interface?

Comment (tatianahutii) [reply frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.html]: yes, real interface

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**